lnst/Controller/ContainerPoolManager.py

```diff
@@ -139,7 +154,7 @@ def _check_machine(self, machine: Machine):
         logging.info(f"Agent process is running at {hostname}")
 
     @staticmethod
-    def _start_container(container: Container, machine: Machine):
```

Comment:
I think you can keep this typehint. The Machine is not lazy imported.

Reply:
same as above